setEnvironment.sh

```diff
@@ -3,21 +3,64 @@
 # Stop at first command failure.
 set -e
 
-if ! [ -f ".env" ]; then
-  echo "Error: .env not found. Can not continue."
-  exit 1
-fi
+# SHOGun paths
+SHOGUN_DIR=../shogun
+SHOGUN_ADMIN_DIR=../shogun-admin
+SHOGUN_CLIENT_DIR=../shogun-demo-client
 
-CURRENT_IP=$(ip route get 1 | awk '{gsub("^.*src ",""); print $1; exit}')
+# Container prefix
+CONTAINER_NAME_PREFIX=shogun
 
+# The mail server host
+MAIL_HOST=mail.terrestris.de
+# The mail server port
+MAIL_PORT=587
+# The password of the mail server
+MAIL_PASSWORD="super secret"
+
+# The database user password
+POSTGRES_USER=shogun
+# The LDAP admin user password
+POSTGRES_PASSWORD=shogun
+
+# The Keycloak host, this must be accessible from inside a docker network and run under HTTPS
+KEYCLOAK_HOST=$(ip route get 1 | awk '{gsub("^.*src ",""); print $1; exit}')
+# The Keycloak admin user
+KEYCLOAK_USER=admin
+# The Keycloak admin password
+KEYCLOAK_PASSWORD=shogun
+
+# The ID of the host user the GS image should run as
 USER_ID=$(id -u)
+# The group ID of the host user the GS image should run as
 GROUP_ID=$(id -g)
+# The name of the host user the GS image should run as
 USER_NAME=$(whoami)
 
-printf "Updating .env:\nKEYCLOAK_HOST=${CURRENT_IP}\nUID=${USER_ID}\nGID=${GROUP_ID}\nUNAME=${USER_NAME}"
+SCRIPT_DIR=$(dirname "${BASH_SOURCE[0]}")
+ENV_FILE=.env
+
+rm -rf $SCRIPT_DIR/$ENV_FILE
```

Comment:
Maybe we should add a small confimr prompt before removing the .env file like
`This will remove the current .env file. Do you want to continue? (y/n)`

Reply:
Good idea, introduced in 6639ef3.